src/components/structures/auth/SetupEncryptionBody.tsx

```diff
@@ -246,23 +253,7 @@ export default class SetupEncryptionBody extends React.Component<IProps, IState>
                     </div>
                 </div>
             );
-        } else if (phase === Phase.ConfirmReset) {
-            return (
-                <div>
-                    <p>{_t("encryption|verification|verify_reset_warning_1")}</p>
-                    <p>{_t("encryption|verification|verify_reset_warning_2")}</p>
-
-                    <div className="mx_CompleteSecurity_actionRow">
-                        <AccessibleButton kind="danger_outline" onClick={this.onResetConfirmClick}>
-                            {_t("encryption|verification|reset_proceed_prompt")}
-                        </AccessibleButton>
-                        <AccessibleButton kind="primary" onClick={this.onResetBackClick}>
-                            {_t("action|go_back")}
-                        </AccessibleButton>
-                    </div>
-                </div>
-            );
-        } else if (phase === Phase.Busy || phase === Phase.Loading) {
+        } else if (phase === Phase.Busy || phase === Phase.Loading || phase == Phase.ConfirmReset) {
```

Comment:
again, bit confused about why it is safe to get rid of the confirm step, and why we now show a spinner for "ConfirmReset".

Reply:
This dialog used to do confirm resetting the identity, but now it uses `ResetIdentityDialog` to do so.  So during the "ConfirmReset" stage, this dialog just shows a spinner, while `ResetIdentityDialog` does the actual confirmation.